exercises/practice/robot-simulator/.meta/example.gd

```diff
@@ -0,0 +1,27 @@
+var _position: Vector2i
+var _dir: String
+var allowed_directions = ['north', 'east', 'south', 'west']
+
+func _init(position: Vector2i = Vector2i(0, 0), direction: String = 'north'):
+	_position = position
+	_direction = direction
+
+
+func move(instructions: String):
+	for instruction in instructions:
+		match [instruction, direction]:
```

Comment:
I think you meant `_direction` here, not `direction` (as the latter will only be available in the constructor).